libs/libsodium/Makefile

```diff
@@ -52,6 +52,7 @@ endef
 
 CONFIGURE_ARGS+= \
 	--disable-ssp \
+	--without-pthreads \
```

Comment:
Why not report bugs to uClibc? I don't have such issue using LEDE trunk with musl-libc.